### DISCUSSION TITLE
Open the browser in split screen orientation

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [spacecowboy/Feeder#246] [Ideas]
I few releases ago the open browser in split screen where added. I find it absolute awesome when reading on a tablet. However I don't seem to understand why and how sometime it opens the new browser on the left or on the right. I prefer when the browser open on the right and keeps Feeder on the left. It would be nice if the user could toggle the _Open the browser in split screen_ feature where to open the new browser, to the left or to the right, (or up or down I suppose).

**Comment by spacecowboy** [comment]
The API does not have a feature to define *where* the windows should be (as far as I know)
